<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormOptions</class>
 <widget class="QMainWindow" name="FormOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>238</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>238</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>11840</width>
    <height>238</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyQtForm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignTop">
     <widget class="QWidget" name="wgTop" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Project Directory</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leProjDir">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>D:\Tuition\Marzena\Year 3\RecalculationsGUI</string>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Sub-Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSubDir">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Forms</string>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QComboBox" name="cbFormType">
         <item>
          <property name="text">
           <string>Dialog with Buttons Bottom</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dialog with Buttons Right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dialog with Buttons</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Main Window</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Widget</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignBottom">
     <widget class="QWidget" name="wgBottom" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbGenerate">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
